src/components/userProfile/components/UserProfileModal.js

```diff
@@ -0,0 +1,135 @@
+        render() {
+            const {
+                form: { getFieldDecorator },
+                visible,
+                onCancel,
+                onCreate,
+                userData: {
+                    address,
+                    city,
+                    country,
+                    email,
+                    firstName,
+                    lastName,
+                    phoneNumber,
+                    username,
+                },
+            } = this.props;
+
+            const PROFILE_INFO = [
+                {
+                    heading: BASIC_INFO_TEXT,
+                    info: [
+                        {
+                            field: 'firstName',
+                            initialValue: firstName,
+                            title: FIRST_NAME,
+                        },
+                        {
+                            field: 'lastName',
+                            initialValue: lastName,
+                            title: LAST_NAME,
+                        },
+                        {
+                            field: 'username',
+                            initialValue: username,
+                            title: USERNAME,
+                        },
+                    ],
+                },
+                {
+                    heading: CONTACT_INFO_TEXT,
+                    info: [
+                        {
+                            field: 'email',
+                            initialValue: email,
+                            title: EMAIL,
+                        },
+                        {
+                            field: 'phoneNumber',
+                            initialValue: phoneNumber,
+                            title: PHONE,
+                        },
+                    ],
+                },
+                {
+                    heading: LOCATION_INFO_TEXT,
+                    info: [
+                        {
+                            field: 'address',
+                            initialValue: address,
+                            title: ADDRESS,
+                        },
+                        {
+                            field: 'city',
+                            initialValue: city,
+                            title: CITY,
+                        },
+                        {
+                            field: 'country',
+                            initialValue: country,
+                            title: COUNTRY,
+                        },
+                    ],
+                },
+            ];
+
+            let inputField;
+            const editProfile = PROFILE_INFO.map(({ heading, info }) => (
+                <div key={heading} className={INFO_DIV}>
+                    <h4 className={HEADER_INFO}>{heading}</h4>
+                    {info.map(({ title, initialValue, field }) => {
+                        if (field === 'email') {
+                            inputField = <input disabled={true} />;
```

Comment:
use `<input disabled />` and why is this being disabled?

Reply:
we don't want users to change their email for now. considering the  auth mechanism we are using, changing the email will see the users as new users.
this will be fixed in our v2. the users can only change their password and other info